java/runtime/src/main/java/io/ray/runtime/RayNativeRuntime.java

```diff
@@ -80,6 +83,40 @@
           tempGcsClient.getInternalConfig().entrySet()) {
         rayConfig.rayletConfigParameters.put(entry.getKey(), entry.getValue());
       }
+
+      // Keep this method logic in sync with `services.get_address_info_from_redis_helper`
+      int numRetries = 5;
+      int retryCount = 0;
+      boolean loadConfig = false;
+      while (retryCount++ < numRetries) {
```

Comment:
We should add a worker type check before reading from GCS. This is only needed for driver process.